Week-4/Session 10 JS/S10_call_apply_bind.js

```diff
@@ -0,0 +1,38 @@
+class Calculator {
+  add(a, b) {
+    return a + b;
+  }
+  subtract(a, b) {
+    return a - b;
+  }
+  multiply(a, b) {
+    return a * b;
+  }
+  divide(a, b) {
+    return a / b;
+  }
+}
+class ScientificCalculator extends Calculator {
```

Comment:
```suggestion

class ScientificCalculator extends Calculator {
```

Reply:
> can you merge latest master to week4 branch first to only have week4 changes? currently previous week changes are also in this PR which shouldn't happen.

beacause I guess I didnt merge week3 code, as it needs refactoring